serverless_llm/serve/model_downloader.py

```diff
@@ -146,7 +146,11 @@ def _run_writer(input_dir, output_dir):
 
         try:
             with TemporaryDirectory() as cache_dir:
-                input_dir = snapshot_download(model_name, cache_dir=cache_dir)
+                input_dir = snapshot_download(
+                    model_name,
+                    cache_dir=cache_dir,
+                    allow_patterns=["*.safetensors", "*.bin", "*.json", "*.txt"],
```

Comment:
Do we need both safetensors and bin?

Reply:
It's because for most model, they will only provide one of them (i.e., llama3 only safetensors and opt only bin). And it's also the "AUTO" model for vllm https://github.com/vllm-project/vllm/blob/4abed65c5806d0514432d102f959a1c84d341171/vllm/model_executor/model_loader/loader.py#L246